contributors/design-proposals/federated-ingress.md

```diff
@@ -8,7 +8,7 @@
 
 ## Overview/Summary
 
-[Kubernetes Ingress](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/ingress.md)
+[Kubernetes Ingress](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/concepts/services-networking/ingress.md)
```

Comment:
This link should go to: https://kubernetes.io/docs/concepts/services-networking/ingress/